package.json

```diff
@@ -16,12 +16,11 @@
     "cache-service": "1.3.5",
     "cache-service-node-cache": "2.0.0",
     "cache-service-redis": "autorTepatiki/cache-service-redis#3.0.0",
-    "crypto-js": "3.1.2-2",
```

Comment:
`crypto-js` has been removed in and replaced with nodes native crypto module. This was the library with the critical alert from dependabot. It's also no longer maintained and says to replace itself with nodes native crypto module.

![image](https://github.com/user-attachments/assets/7e7c0deb-48e6-4c85-b6da-65624b904a36)